scripts/mail-alarm

```diff
@@ -277,20 +350,29 @@ class Dom0LogFSUsageAlarmETG(EmailTextGenerator):
 
     def generate_subject(self):
         pool_name = get_pool_name()
-        return '[%s] %s Alarm: Log partition nearly full on "%s"' % (pool_name,
+        subject_str = self.mail_subject()
+        if subject_str is None:
+            subject_str = '[%s] %s Alarm: Log partition nearly full on "%s"'
+
+        return subject_str % (pool_name,
                    branding.PRODUCT_BRAND, self.params['name_label'])
 
     def generate_body(self):
-        return \
-            'The log partition usage on "%s" is at %.1f%%.\n' \
+        body_str = self.mail_body()
+        if body_str is None:
+            body_str = 'The log partition usage on "%s" is at %.1f%%.\n' \
             'This alarm is set to be triggered when log partition usage is more than %.1f%%.\n' \
-            '\n' % \
+            '\n'
+
+        return \
+            body_str % \
             (self.params['name_label'],
              self.value * 100.0,
              self.alarm_trigger_level * 100.0)
 
 class Dom0MemUsageAlarmETG(EmailTextGenerator):
```

Comment:
This name is counter-intuitive imho, but it's safer to stick with the name we've got.